_data/simple-icons.json

```diff
@@ -4389,8 +4389,9 @@
         },
         {
             "title": "DocuSign",
-            "hex": "FFCC22",
-            "source": "https://www.docusign.com/sites/all/themes/custom/docusign/favicons/mstile-310x310.png"
+            "hex": "4D00FC",
```

Comment:
Where did you get this color from?

Reply:
(I'm seeing `#4c00ff` from the color section of their brand guidelines.)

Reply:
cannot remember where i copied. just updated following their brand palette